examples/notebooks/PastingLawForPullbacks.ipynb

```diff
@@ -176,7 +176,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "58 primitive operations were used to derive 277 operations for this category which algorithmically\n",
+      "58 primitive operations were used to derive 267 operations for this category which algorithmically\n",
```

Comment:
This should not have happened. It seems like that the notebooks in the CI still do not always prefer the CAP from the GAP user root.

Reply:
Exactly, I ran into this problem on the `functional` branch of `FunctorCategories`: `SkeletalFinSets` had no (co)limits. This prompted me to add the suggested dependency on `CategoryConstructor` back to `Toposes`. And yes, making a user interaction necessary is undesirable.